src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/services/jobservices/CleanUpMessageJobTest.java

```diff
@@ -115,10 +116,20 @@ void should_clean_messages_when_environment_is_non_prod(CapturedOutput output) {
 
         String expectedLogMessage = "CleanUpJobConfiguration(environment=aat, deleteLimit=5, startedDaysBefore=90, "
                                     + "stateForProd=[PROCESSED], stateForNonProd=[PROCESSED, READY])";
+        assertConsoleOutputHasMessages(output, expectedLogMessage);
+    }
 
-        assertTrue(output.getOut().contains(expectedLogMessage));
-        assertTrue(output.getOut().contains("CLEAN_UP_MESSAGES job completed"));
+    private void assertConsoleOutputHasMessages(CapturedOutput output, String expectedLogMessage) {
+        await().ignoreException(Exception.class)
+            .pollInterval(100, MILLISECONDS)
+            .atMost(5, SECONDS)
+            .untilAsserted(() -> {
+                assertTrue(output.getOut().contains(expectedLogMessage));
```

Comment:
Looks like a good time to also make the assertions consistent using Assertions.assert?? (Junit5)

Reply:
updated